Exercise_D and E/index.html

```diff
@@ -15,8 +15,30 @@
     />
   </head>
   <body>
-    <!-- CODE HERE -->
-
+    <img src="assets/images/logo.png">
+    <Div class=Top>
+      <h2>Beautiful Flower</h2>
+      <button type="button">Learn More</button>
+      
+    </Div>
+    <Div class=below>
+      <Div class=four>
+        <img src="./assets/images/flower-icon.png">
```

Comment:
Come back and alter this so that you can get the icons onto two lines 


Your code is decent, try to hard code less and follow conventions. 
Do not capitalise HTML elements